diff --git a/pyznap/utils.py b/pyznap/utils.py
index 1182419..116f23a 100644
--- a/pyznap/utils.py
+++ b/pyznap/utils.py
@@ -76,7 +76,7 @@ def read_config(path):
         logger.error('Error while loading config: File {:s} does not exist.'.format(path))
         return None
 
-    parser = ConfigParser()
+    parser = ConfigParser(inline_comment_prefixes="#")
     try:
         parser.read(path)
     except (MissingSectionHeaderError, DuplicateSectionError, DuplicateOptionError) as e:
